.github/workflows/publish-release.yml

```diff
@@ -54,4 +57,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
           curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
-          bin/hub pull-request --base main -m "[Automated] Sync master after "$VERSION" release"
+          bin/hub pull-request -m "[Automated] Sync master after "$VERSION" release"
```

Comment:
Any reason to remove this property? This is needed if the main branch is "main"

Reply:
Yeah it's a mistake. Reverted the change in afcc3401f57986ca5862645c76acf379977ac8be